wpa_supplicant/config.c

```diff
@@ -5597,6 +5598,7 @@ static const struct global_parse_data global_fields[] = {
 	/* NOTE: When adding new parameters here, add_interface() in
 	 * wpa_supplicant/dbus_new_introspect.c may need to be modified to
 	 * increase the size of the iface->xml buffer. */
```

Comment:
what about this comment? I don't see a corresponding change in`wpa_supplicant/dbus_new_introspect.c` is that not needed?